scenarios/02_Fighting_for_Passage.cfg

```diff
@@ -489,6 +555,14 @@
         [/teleport]
         {CLEAR_VARIABLE XOEAME_TMP}
         {CLEAR_VARIABLE YOEAME_TMP}
+
+        [unhide_unit]
+            id=Oeame
+        [/unhide_unit]
+        [redraw][/redraw]
+        [delay]
+            time=10
```

Comment:
Time=10 is really short, suggest something closer to 50 or more.

Reply:
ok, changed to 55 in 61ba872